Reword PR Template (#700)
python-discord/sir-lancebot: .github/pull_request_template.md

## Relevant Issues
<!--

> Linking an issue approved by a Core Developer, which will have the "approved" label, is mandatory.

Reviewer: I had to read this sentence twice to understand it, does this read better? 
```suggestion
Linking an issue approved by a Core Developer, that has the "approved" label, is mandatory.
```
Maybe the commas aren't necessary too

Reviewer 2: Tbh the wording sounds a bit unnatural in general.

> It is mandatory to link an issue with the "approved" label. 

We don't really need to outline that it's approved by a Core Dev, when the label already says that in it's description, do we?

Author: The reason for adding it in the first place was to address concerns about who is approving issues. Currently, anyone with write permissions (all staff members) can add the approved label. The conversation was linked in the issue, but [here](https://discord.com/channels/267624335836053506/411200599653351425/832591634952618004) it is again (staff channel). I don't mind it either way, but the consensus at the time was approving issues is a core dev responsibility, we should outline that.

Reviewer 2: Okay. In that case, it's really only getting rid of the inner phrasing to make it flow better while keeping the same info.

> It is mandatory to link to an issue that has been approved by a Core Developer, indicated by an "approved" label.

Author: Addressed in 525078b

Issues can be skipped with explicit core dev approval, but you have to link the discussion.
-->

<!-- Link the issue by typing: "Closes #<number>" (Closes #0 to close issue 0 for example). -->

## Description
<!-- Describe what changes you made, and how you've implemented them. -->

## Did you:
<!-- These are required when contributing. -->
<!-- Replace [ ] with [x] to mark items as done. -->

- [ ] Join the [**Python Discord Community**](https://discord.gg/python)?

> - [ ] Read all the comments in this tempelate?

Reviewer 3: How will we be encouraging linting, if at all?

Reviewer: It is mentioned inside the contrib guidelines. It may be worth to mention them here again but in the end the failed lint will be the same thing, we’ll end up with a second fix commit.

Reviewer 3: Maybe we should link the contributing guidelines in a comment within this PR template?

Reviewer: It is already mentioned line 20 but I feel like it could even be the first line of the template. We could also add a sentence about when this template should be used.

Reviewer 3: I somehow completely missed that link with my eyes. Sorry about that, this PR looks good to me.
